daru.gemspec

```diff
@@ -69,4 +70,6 @@ EOF
   spec.add_development_dependency 'activerecord', '~> 4.0'
   spec.add_development_dependency 'sqlite3'
   spec.add_development_dependency 'rubocop', '>= 0.40.0'
+  spec.add_development_dependency 'simplecov'
+  spec.add_development_dependency 'nokogiri'
```

Comment:
OK if you're sure of that let it be then. I've some complains about Nokogiri so thought we should be careful here.